integration-cli/docker_cli_run_test.go

```diff
@@ -3297,7 +3297,7 @@ func (s *DockerSuite) TestRunContainerNetModeWithExposePort(c *check.C) {
 	}
 
 	out, _, err = dockerCmdWithError("run", "--expose", "5000", "--net=container:parent", "busybox")
-	if err == nil || !strings.Contains(out, "Conflicting options: --expose and the network mode (--expose)") {
+	if err == nil || !strings.Contains(out, "Conflicting options: --expose and the network mode (--net)") {
```

Comment:
Why this change?  And, are you not able to use the GetErrorMessage(...) trick you use in the previous test file?